celery/readme.txt

```diff
@@ -1 +0,0 @@
-This directory is used for celery beat scheduler to write celerybeat-schedule file for scheduling purposes.
```

Comment:
@pkdash @mauriel133 Please don't remove this file but keep this readme.txt file there for informational purpose. This is the only comment I have for this branch now. After this is addressed, I can +1